rpcs3/util/media_utils.cpp

```diff
@@ -358,14 +358,14 @@ namespace utils
 			}
 
 			duration_ms = stream->duration / 1000;
-			
-			AVPacket packet{};
-			av_init_packet(&packet);
+
+			AVPacket* packet = av_packet_alloc();
```

Comment:
Why was this changed and why doesn't it check for null?

Reply:
Idk, it's probably null only on out of memory, which itself is very serious problem which can't be worked around easily.